external/fv3fit/fv3fit/keras/_models/convolutional.py

```diff
@@ -91,21 +90,15 @@ def train_convolutional_model(
         train_data = tuple(train_data)
     X, y = train_data[0]
     train_model, predict_model = build_model(hyperparameters, X=X, y=y)
-    output_metadata = get_stacked_metadata(
-        names=hyperparameters.output_variables,
-        ds=train_batches[0],
-        unstacked_dims=UNSTACKED_DIMS,
-    )
     del train_batches
     hyperparameters.training_loop.fit_loop(
         model=train_model, Xy=train_data, validation_data=validation_data
     )
     predictor = PureKerasModel(
         input_variables=hyperparameters.input_variables,
         output_variables=hyperparameters.output_variables,
-        output_metadata=output_metadata,
         model=predict_model,
-        unstacked_dims=("x", "y", "z", "z_interface"),
+        unstacked_dims=("x", "y", "z"),
```

Comment:
Were there any variables on the z_interface dimension we would want to use to begin with? I can't think of any.

Reply:
I don't think so. Even if we made a flux-based model that internally produces cell-interface variables, we'd still convert those to cell-center tendencies before making them outputs of the model. If this becomes an issue we can add that functionality when we need it.